### ISSUE TITLE
keras autoregression part 1 error. STUDENT COPY

### ISSUE THREAD

**Issue by TomMonks** [health-data-science-OR/forecasting#36]
There are some issues in the STUDENT notebook for autoregression using KERAS.

1. The network dealing with the noisy cosine wave is unstable. This can be fixed in the solutions notebook by a higher learning rate. This does not seem to fix instability in the student copy

2. There is an issue with the direct model trying to predict test data of the wrong shape.  This is because the test data used is from a train test split EARLIER in the notebook. that earlier split uses a different WINDOW_SIZE.  This is solved by conducting a train test split AGAIN before prediction.  Easy fix.

Recommend that the solution notebook is modified and then copied over the student notebook to ensure they are both consistent.

**Comment by TomMonks** [comment]
Point 1 fixed in version 5.1.0.   Point 2 did not seem to be an issue. Potentially fixed earlier without a note.
